Library/Formula/agda.rb

```diff
@@ -17,29 +15,38 @@ class Agda < Formula
     sha256 "d0e17808ee896cdedc499b010b67961dc3db3c430c501faef16c4b4482238447" => :mountain_lion
   end
 
-  head "https://github.com/agda/agda.git", :branch => "master"
+  head do
+    url "https://github.com/agda/agda.git", :branch => "master"
+    resource "stdlib" do
+      url "https://github.com/agda/agda-stdlib.git", :branch => "master"
+    end
+  end
+
+  resource "stdlib" do
+    url "https://github.com/agda/agda-stdlib.git", :branch => "2.4.2.4"
```

Comment:
I understand.  However, the Agda standard library is explicitly [described as experimental](https://github.com/agda/agda-stdlib/blob/master/CHANGELOG#L7-L8).  Today, the `2.4.2.4` branch has [received changes](https://github.com/agda/agda-stdlib/commits/2.4.2.4) backported from `master`.